examples/11_burn_native_token_without_foundry.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! cargo run --example 11_burn_native_token_without_foundry --release
```

Comment:
If we have the clear separation from burning/melting (only possible with foundry) I think we don't need to add the `without_foundry` to the name

Reply:
Exactly!